### DISCUSSION TITLE
Please add Identity

### DISCUSSION THREAD

**Discussion by amirmehdi76** [jasontaylordev/CleanArchitecture#812] [Ideas]
Can you add Identity with Role And RoleClaims ?

**Comment by hsandhar** [comment]
@amirmehdi76 Perhaps I misunderstood your question, but there's already Identity and Role/Policy setup/examples in the project.

<img width="1466" alt="image" src="https://user-images.githubusercontent.com/1728547/222979867-e6fe18bf-e44d-4c39-b9e9-5f25575cbd85.png">
